### ISSUE TITLE
Citizen management ignores tiles claimed by other cities

### ISSUE THREAD

**Issue by CodeKatta** [yairm210/Unciv#7465]
**Version**
4.1.18 and older

**Describe the bug**
Citizen management doesn't use tiles which have already been claimed by a neighboring city although that city doesn't use that tile.
Since it's not in use by the original owner it can be manually assigned to the production of the other city but whenever some new event occurs (new tile gets incorporated, production finished, etc.) the citizen management yanks everything off that tile again (except of course it's locked).
This even occurs if that tile is outside the first city's 3-tile-radius (3TR). This can happen if that field contains some kind of resources and the city in question already owns most of the tiles within its 3TR. However, since it's outside its 3TR it can't use it but the neighboring city could but doesn't do so voluntarily.
So would it be possible to simply reassign the ownership of tiles? In other words, whoever can access a tile and uses it owns it.

**Comment by OptimizedForDensity** [comment]
> This even occurs if that tile is outside the first city's 3-tile-radius (3TR). This can happen if that field contains some kind of resources and the city in question already owns most of the tiles within its 3TR. However, since it's outside its 3TR it can't use it but the neighboring city could but doesn't do so voluntarily.

You mean a resource that is owned by a city more than 3 tiles away but 3 or fewer tiles away from a non-owning city? In that case, it sounds like the closer city should be able to work that tile IMO (and city AI updated to use those tiles) unless Civ 5 doesn't let that happen.

> So would it be possible to simply reassign the ownership of tiles? In other words, whoever can access a tile and uses it owns it.

This wouldn't work since the culture/gold cost for buying/expanding to a new tile is based on the number of tiles currently owned by the city. If cities can exchange tiles freely, then a player could exploit this by transferring all the tiles to another city and buying new territory for cheap.

**Comment by CodeKatta** [comment]
> You mean a resource that is owned by a city more than 3 tiles away but 3 or fewer tiles away from a non-owning city?

Yes, for instance (as the most extreme case).

> In that case, it sounds like the closer city should be able to work that tile IMO (and city AI updated to use those tiles if they're unused) unless Civ 5 doesn't let that happen.

That's what I said; it is possible to work that tile but you have to assign it manually. However, if you click on 'XYZ Focus' the citizen management AI moves production from that tile to some other tile even if that particular tile contains a big fat XYZ bonus. It's simply treated as if it were not within your borders.

> This wouldn't work since the culture/gold cost for buying/expanding to a new tile is based on the number of tiles currently owned by the city. If cities can exchange tiles freely, then a player could exploit this by transferring all the tiles to another city and buying new territory for cheap.

Well, I couldn't care less who owns the tile (you could make the transfer cost a certain amount of gold). It's just that the citizen management AI doesn't recognize that this tile is workable although it is
a) within your borders
b) within the city's 3 tile radius
c) not in use by the other city

And since it's not just some run-of-the-mill tile with low value but usually one with high production/food/gold output, the automation in such a city is pretty much useless.

I just thought that a transfer might make it easier for the citizen management AI to recognize that a free tile is usable. But if there's a more elegant solution that's even better.

**Comment by itanasi** [comment]
The problem is that which city owns which tiles is very important to maintain due to knowing which tiles go with the city should it get captured. So transferring the tiles between cities in the underlying structure isn't a good solution.

However, there may be some better logic so that the automation will look for tiles owned by another city but not being worked by the other city.

**Comment by CodeKatta** [comment]
> This distinction between owned & workable must already exist, since the UI is able to do it.

Yes, I had that impression as well, since the citizen management algorithm most certainly does recognize if a tile is 'owned & non-workable' because the other city already occupies that tile. At least it doesn't just kick the other city's production off that field.
